src/daos/daos/daos_erase.sh

```diff
@@ -1,15 +1,21 @@
 #!/bin/bash
 
 source $CEPH_PATH/src/daos/require_variables.sh
+source $CEPH_PATH/src/daos/wait_for.sh
 
-daos_erase()
+function daos_wait_erase()
```

Comment:
Since this does not include erase, it could be renamed to `daos_wait_query` or something like that, which could be reused

Reply:
here is the erase command: **$COMMAND_PREFIX ${DAOS_BIN}/dmg system erase**.  That method is specific to daos_erase() and would have been defined as static if it was in C++.  Its waiting for the erase status to change and is passed to and called by the wait_for function.  I disagree with renaming it